sys-apps/groff/groff-1.22.4.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-inherit toolchain-funcs
+inherit autotools toolchain-funcs
```

Comment:
Okay, so just to clarify, I have been reading some ebuilds and it looks like since we patch the generated makefile here, the correct order in `src_prepare` would be `default`->`eautoreconf`->fixes?  As opposed to ebuilds which modify the handmade files such as `Makefile.am` which use the order `default`->fixes->`eautoreconf`?  I hope that makes sense, I just want to make sure that the correct location to call it should be directly under `default` in `src_prepare`?

Reply:
well, default includes the opportunity for users to apply their patches via /etc/portage/patches, so if user wants to apply some patches that needs scripts regeneration it is good to have the default before eautoreconf

Reply:
Alright, so my understanding is it should be done like so:  https://github.com/gentoo/gentoo/pull/10697/commits/d89779acf4c486600845126b4bea32516530ef23